packages/experiments/src/components/CollapsibleSection/CollapsibleSection.ts

```diff
@@ -21,3 +21,13 @@ export const CollapsibleSection: React.StatelessComponent<ICollapsibleSectionPro
   },
   CollapsibleSectionState
 );
+
+// TODO: This is only here for testing createComponent and should be removed before promoting to production
+export const CollapsibleSectionStateless: React.StatelessComponent<ICollapsibleSectionProps> = createStatelessComponent<
+  ICollapsibleSectionViewProps,
+  ICollapsibleSectionStyles
+>({
+  displayName: 'CollapsibleSection',
+  view: CollapsibleSectionView,
+  styles
```

Comment:
customizableFields: ['foo', 'bar', 'baz']; maybe

Reply:
Yep, that's what I was thinking, assuming we still needed to pipe them in (tied to other comment)